migrations/deploy/Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM schemahero/schemahero:0.12.0
 
+USER root
+RUN apt-get update && apt-get install -y --only-upgrade --no-install-recommends \
```

Comment:
this won't upgrade packages though, right?

Reply:
yeah, it upgrades the two on the next line `passwd login`